app/controllers/tasks_controller.rb

```diff
@@ -25,8 +41,8 @@ def create
   def edit; end
 
   def update
-    if @task.update(task_params)
-      redirect_to @task, notice: 'The task has been updated!'
+    if update_task.task
```

Comment:
`task` is always present in the context, it means that this expression is always `true`. Thus this `if` statement is meaningless